src/shims/android_stubs.h

```diff
@@ -20,4 +20,8 @@
 #define _Static_assert(...)
 #endif
 
+#ifndef TRASHIT
```

Comment:
I'm fairly sure this has been moved to `internal.h` already in a previous PR, please remove.